source/content/guides/integrated-composer/02-dependencies.md

```diff
@@ -19,9 +19,7 @@ It's important to understand how `require` and `require dev` are used on the Pan
 
 ### Composer `require` Section
 
-You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. Drupal modules / themes and WordPress plugins / themes should always be in the `require` section, not the `require-dev` section. All dependencies in the `require` section are pushed to Pantheon Dev and Multidev environments, but not to Test and Live environments.
-
-You do not need to pass the `--no-dev` option to prevent dependencies from being pushed to your Test and Live environments. This option is passed automatically for Drupal modules / themes and WordPress plugins / themes in the `require` section.
```

Comment:
@greg-1-anderson Can the --no-dev option content be moved to the require dev section? We have users specifically searching for the term --no-dev on the Docs site.